app/scheme.go

```diff
@@ -127,11 +127,11 @@ func (a *App) GetTeamsForScheme(scheme *model.Scheme, offset int, limit int) ([]
 		return nil, err
 	}
 
-	result := <-a.Srv.Store.Team().GetTeamsByScheme(scheme.Id, offset, limit)
-	if result.Err != nil {
-		return nil, result.Err
+	teams, err := a.Srv.Store.Team().GetTeamsByScheme(scheme.Id, offset, limit)
```

Comment:
Probably is simpler here just `return a.Srv.Store.Team().GetTeamsByScheme(scheme.Id, offset, limit)`